pom.xml

```diff
@@ -81,13 +82,14 @@
 
     <!-- Test only dependencies -->
     <junit.version>4.13.1</junit.version>
-    <junit-jupiter.version>5.7.0</junit-jupiter.version>
+    <junit-jupiter.version>5.9.0</junit-jupiter.version>
     <mockito.version>3.6.28</mockito.version>
     <assertj.version>3.18.1</assertj.version>
     <awaitility.version>4.0.3</awaitility.version>
     <hamcrest.version>1.3</hamcrest.version>
     <testcontainers.version>1.15.1</testcontainers.version>
     <okhttp.version>4.9.0</okhttp.version>
+    <okhttp5.version>5.0.0-alpha.10</okhttp5.version>
```

Comment:
Do we want to support alphas? cc @llinder

Reply:
currently, okhttp5 only has alpha version, Since we're only using it for testing, I think it's still acceptable